proto/org/hypertrace/agent/config/v1/config.proto

```diff
@@ -98,28 +98,32 @@ message DataCapture {
 
 // PropagationFormat represents the propagation formats supported by agents
 enum PropagationFormat {
+
+    // Default to none. Agent will use its default propagation format
+    PROPAGATION_FORMAT_UNSPECIFIED = 0;
```

Comment:
The `PROPAGATION_FORMAT_` prefix is a little heavy for user entered config.  Maybe we should add a lint exception for these enums.

Though @jcchavezs @pavolloffay

Reply:
Yes please, I think it is way too much verbosity.